Resources/Prototypes/_Moffstation/PersonalItems/Wearables/FriedFCC/Fred_Neuman/fredwatch.yml

```
@@ -0,0 +1,30 @@
+# Player: FriedFCC - Character: Fred Neuman
+- type: entity
+  parent: WristwatchGold
+  id: PersonalItemFredWatch
+  name: Fred's ancient watch
+  description: A fancy looking watch that looks like it was made hundreds of years ago. Despite it's age it's in pristine condition.
```

Despite *it is* age, it is in pristine condition.
```suggestion
  description: A fancy looking watch that looks like it was made hundreds of years ago. Despite its age it's in pristine condition.
```